base/src/main/java/io/spine/code/proto/AbstractMessageDeclaration.java

```diff
@@ -69,6 +72,22 @@ public FileDescriptorProto getFile() {
         return file;
     }
 
+    public MessageDocumentation documentation() {
+        return new MessageDocumentation(this);
+    }
+
+    /**
+     * Obtains declarations of the message fields.
+     *
+     * @return fields in the order of their declaration
+     */
+    public List<FieldDeclaration> fields() {
```

Comment:
I think it should be `ImmutableList`. We aim for clarity. There is a related [reading](http://errorprone.info/bugpattern/MutableConstantField).